ocaml/xapi/xapi_db_upgrade.ml

```diff
@@ -868,6 +868,18 @@ let remove_legacy_ssl_support =
       )
   }
 
+let upgrade_last_updated_field =
+  {
+    description= "update Host.last_software_update field"
+  ; version= (fun _ -> true)
```

Comment:
Let do
```suggestion
  ; version= (fun x -> x <= yangtze)
```
so that it doesn't kick in on future upgrades, where the field is set in a different place.